ci-operator/jobs/openshift/cluster-network-operator/openshift-cluster-network-operator-release-4.8-presubmits.yaml

```diff
@@ -282,6 +282,7 @@ presubmits:
       ci-operator.openshift.io/prowgen-controlled: "true"
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-cluster-network-operator-release-4.8-e2e-azure-ovn-dualstack
+    optional: true
```

Comment:
also, change always_run to false, Line 272

Reply:
Sure, thanks!